imap_processing/utils.py

```diff
@@ -101,17 +101,22 @@ def convert_raw_to_eu(dataset: xr.Dataset, conversion_table_path, packet_name):
     return dataset
 
 
-def create_dataset(packets: list[Packet], met_name="shcoarse") -> xr.Dataset:
+def create_dataset(
+    packets: list[Packet], met_name="shcoarse", include_header=True, skip_keys=None
+) -> xr.Dataset:
     """Create dataset for each metadata field.
 
-    # TODO: change IMAP-Hi to use this and others as needed.
-
     Parameters
     ----------
     packets : list[Packet]
         packet list
-    met_name : str
-        metadata name to use as epoch time
+    met_name : str, Optional
```

Comment:
Do some packets contain a value that is valid to directly use as the epoch coordinate? The few examples that I have seen require a function that converts what is the packet to what is desired to be stored in the epoch coordinate. For example, a SPICE call to convert CCSDS_MET to a datetime object?

Reply:
In this phase, I am not converting because some uses this exact key to sort data by this time. May be we should change time here in the format the CDF desire(datetime64 object). I could do that in upcoming PR.